shell/browser/ui/cocoa/atom_menu_controller.mm

```diff
@@ -193,8 +193,18 @@ - (void)moveMenuItems:(NSMenu*)source to:(NSMenu*)destination {
 // Replaces the item's submenu instance with the singleton recent documents
 // menu. Previously replaced menu items will be recovered.
 - (void)replaceSubmenuShowingRecentDocuments:(NSMenuItem*)item {
-  NSMenu* recentDocumentsMenu =
-      [[[recentDocumentsMenuItem_ submenu] retain] autorelease];
+  NSMenu* recentDocumentsMenu = nil;
+  if ([recentDocumentsMenuItem_ submenu]) {
+    recentDocumentsMenu =
+        [[[recentDocumentsMenuItem_ submenu] retain] autorelease];
+  } else {
+    recentDocumentsMenu = [[NSMenu alloc] initWithTitle:@""];
+    NSMenuItem* clearItem =
+        [[NSMenuItem alloc] initWithTitle:@"Clear Menu"
```

Comment:
Does this need to be localized?

Reply:
not particularly :)

Reply:
Will this show in English to users whose system language is set to a non-English language?

Reply:
ah great catch, ty 🙇‍♀